core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -37,10 +37,150 @@
+        </div>
+        <div class="conversation-skin-right-card-container" ng-show="isCurrentSupplementalCardNonempty()">
+        
+        
+        
+                          <div class="conversation-skin-supplemental-interaction-container">
+                              <div class="conversation-skin-help-card" ng-if="helpCardHtml">
+                                <img class="conversation-skin-oppia-avatar"
+                                     ng-src="<[OPPIA_AVATAR_IMAGE_URL]>" alt="">
+                                <button type="button" class="conversation-skin-close-help-card-button"
+                                        ng-click="clearHelpCard()" ng-if="!helpCardHasContinueButton">
+                                  <i class="material-icons md-18">&#xE5CD;</i>
+                                </button>
+                                <div class="conversation-skin-help-card-content" angular-html-bind="helpCardHtml">
+                                </div>
+                                <br>
+                                <md-button class="oppia-learner-continue-button protractor-test-continue-to-next-card-button"
+                                   focus-on="<[CONTINUE_BUTTON_FOCUS_LABEL]>"
+                                   ng-click="showPendingCard(upcomingStateName, upcomingParams, upcomingContentHtml)"
+                                   ng-if="helpCardHasContinueButton"
+                                   style="margin-top: 12px;">
+                                  Continue to next card
+                                </md-button>
+                              </div>
+
+                              <div ng-if="!exploration.isInteractionInline(activeCard.stateName)">
+                                <div class="protractor-test-conversation-input"
+                                     angular-html-bind="activeCard.interactionHtml">
+                                </div>
+                              </div>
+                            </div>
+
+                            <div ng-if="!canWindowFitTwoCards()">
+                              <md-button class="oppia-learner-continue-button" ng-click="setVisiblePanel(PANEL_TUTOR)">
+                                <i class="material-icons md-18" style="position: relative; top: 3px;">&#xE5C4;</i>
+                                Instructions
+                              </md-button>
+                            </div>
+        
+        
+        
+        
+        </div>
+      </div>
       <div class="conversation-skin-main-tutor-card conversation-skin-tutor-card-active"
           ng-class="{'conversation-skin-left-card': isCurrentSupplementalCardNonempty() && canWindowFitTwoCards(), 'conversation-skin-has-subsidiary-cards': isCurrentSupplementalCardNonempty() && !canWindowFitTwoCards(),
             'animate-card-width': startCardChangeAnimation}"
-          ng-show="isPanelVisible(PANEL_TUTOR)">
+          ng-show="false && isPanelVisible(PANEL_TUTOR)">
```

Comment:
This is just for testing - In this way I can test only condition with the entire ">= 960px"